pom.xml

```diff
@@ -77,27 +77,11 @@
         <dependencies>
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
-                <artifactId>bom-2.361.x</artifactId>
-                <version>2102.v854b_fec19c92</version>
+                <artifactId>bom-2.387.x</artifactId>
+                <version>2378.v3e03930028f2</version>
```

Comment:
Picking up https://github.com/jenkinsci/workflow-cps-plugin/pull/759, without which the new test exhibits different problems.